scripts/iiab-diagnostics

```diff
@@ -98,6 +98,18 @@ echo "/usr/bin/iiab-gen-iptable"
 cat /usr/bin/iiab-gen-iptables
 echo
 echo "=========================================================="
+echo "/.iiab-image"
+cat /.iiab-image
```

Comment:
`cat /.iiab-image` command will fail when `/.iiab-image` does not exist, as is often the case.

Wrap it in a bash if statement presumably?